db/migrate/20201217034315_create_availabilities.rb

```diff
@@ -1,9 +1,9 @@
 class CreateAvailabilities < ActiveRecord::Migration[6.1]
   def change
     create_table :availabilities do |t|
-      t.references :users, null: false, foreign_key: true
-      t.datetime :start_date_time
-      t.datetime :end_date_time
+      t.references :user, null: false, foreign_key: true
+      t.integer :start_date_time, :limit => 8
+      t.integer :end_date_time, :limit => 8
```

Comment:
Was getting errors that some datetimes were too large so I increased the limit. The data type in schema is now a bigint.